client/components/Coverages/CoverageEditor/CoverageForm.tsx

```diff
@@ -310,6 +311,69 @@ export class CoverageFormComponent extends React.Component<IProps, IState> {
         }
     }
 
+    onAddToWorkflowChange() {
+        const {vocabulary} = superdeskApi.entities;
+        const {gettext} = superdeskApi.localization;
+        const coverageStatuses = vocabulary
+            .getAll()
+            .get('newscoveragestatus').items as Array<IPlanningNewsCoverageStatus>;
+        const updatedCoverage = planningUtils.addCoverageToWorkflow(this.props.value, coverageStatuses);
+        const filteredCoverages = this.props.coverages.filter((x) => x.coverage_id !== updatedCoverage.coverage_id);
+        const {workflow_status, news_coverage_status, assigned_to, add_coverage_to_workflow} = updatedCoverage;
+
+        if (updatedCoverage.add_coverage_to_workflow) {
+            this.props.onChange(
+                'coverages',
+                [
+                    ...filteredCoverages,
+                    {
+                        ...this.props.value,
+                        workflow_status: workflow_status,
+                        news_coverage_status: news_coverage_status,
+                        assigned_to: {
+                            ...assigned_to,
+                            state: assigned_to.state,
+                        },
+                        add_coverage_to_workflow: add_coverage_to_workflow,
+                    },
+                ],
+            );
+        } else {
+            showModal(({closeModal}) => (
+                <Modal
+                    visible
+                    headerTemplate={gettext('Are you sure?')}
```

Comment:
We have a `confirm` method in superdesk-api.d.ts that you could use here. Also it's a good UX practice to phrase modal titles in such a way so it's clear what the modal is about, even if we assume that user forgot which action he clicked to iniate the modal. It also helps to reassure users that they are indeed confirming the intended thing.